packages/core/src/components/DialogBox/types.ts

```diff
@@ -14,6 +14,8 @@ export interface DialogBoxProps extends HTMLProps<HTMLDivElement> {
     minHeight?: string;
     /** Set it true to close the modal on click outside of popup */
     shouldCloseOnOutsideClick?: boolean;
+    /** Set it true to show the close(X) icon on top right corner */
+    showCloseIcon?: boolean;
```

Comment:
I think `withCloseIcon` would be better naming convention, but that's not something I feel very strongly about. Feel free to keep it as it is.